packages/vite/src/node/plugins/index.ts

```diff
@@ -62,8 +62,8 @@ export async function resolvePlugins(
     wasmPlugin(config),
     webWorkerPlugin(config),
     workerImportMetaUrlPlugin(config),
-    assetPlugin(config),
     ...normalPlugins,
+    assetPlugin(config),
```

Comment:
I'm a bit concern of changing this. It could be a breaking change as some Vite plugins expect the asset plugin to process something first before reaching to them. Is it possible to revert this? Or is it needed in this PR

Reply:
`renderChunk` in rollup is use plugins order?
because assets plugin use this hook process `/__VITE_ASSET__([a-z\d]{8})__(?:\$_(.*?)__)?/g` strings to filename.

Reply:
`fileToUrl ` will transform assets url to `__VITE_ASSET__${contentHash}__${postfix ? `$_${postfix}__` : ``}`

Reply:
It seems that really don't need to be in this pr 🙈

Reply:
Hmm. Perhaps if someone were to use `fileToUrl` it has to be a pre plugin then. The asset plugin has `resolveId` and `load` hooks that make the re-ordering risky.

Reply:
user got a url strings in `renderChunks` is greet!

Reply:
> Hmm. Perhaps if someone were to use `fileToUrl` it has to be a pre plugin then. The asset plugin has `resolveId` and `load` hooks that make the re-ordering risky.

maybe not? `fileToUrl` just need `renderChunks` to replace `__VITE_ASSET__${contentHash}__${postfix ? $_${postfix}__ : ``}` to actual url